build-configs/aschaffenburg/index.js

```diff
@@ -14,6 +14,7 @@ const APPLE_APP_ID = '1551810291'
 
 const commonAschaffenburgBuildConfig: CommonBuildConfigType = {
   appName: 'Aschaffenburg App', // TODO maybe we have to change this to 'AB App'
+  appDescription: 'App of the city Aschaffenburg to inform about social topics.',
```

Comment:
Where did you get this from? Probably this has to be changed later, right? If so, please add a TODO (maybe in https://issues.tuerantuer.org/browse/IGAPP-453?)

Reply:
added it to the description of #453